diff --git a/CTS-BE.csproj b/CTS-BE.csproj
index 922a647..5a07aa3 100644
--- a/CTS-BE.csproj
+++ b/CTS-BE.csproj
@@ -26,34 +26,31 @@
       <!-- <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory> -->
       <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
     </None>
-    <FrameworkReference Include="Microsoft.AspNetCore.App" />
     <PackageReference Include="AutoMapper.Extensions.Microsoft.DependencyInjection" Version="12.0.1" />
     <PackageReference Include="Microsoft.AspNet.SignalR" Version="1.0.1" />
     <PackageReference Include="Microsoft.AspNet.WebApi.Client" Version="6.0.0" />
     <PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="6.0.4" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="7.0.5" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="7.0.5">
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="7.0.20" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="7.0.20">
         <PrivateAssets>all</PrivateAssets>
         <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="7.0.5">
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="7.0.20">
         <PrivateAssets>all</PrivateAssets>
         <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
     <PackageReference Include="MSTest.TestFramework" Version="3.2.2" />
-    <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="7.0.4" />
     <PackageReference Include="Swashbuckle.AspNetCore" Version="6.2.3" />
     <PackageReference Include="itext7.pdfhtml" Version="4.0.5" />
     <PackageReference Include="Microsoft.AspNetCore.SignalR" Version="1.1.0" />
     <PackageReference Include="NPOI" Version="2.6.0" />
     <PackageReference Include="Scriban" Version="5.7.0" />
     <PackageReference Include="xunit.extensibility.execution" Version="2.7.0" />
-    <PackageReference Include="AutoMapper.Extensions.Microsoft.DependencyInjection" Version="12.0.1" />
     <PackageReference Include="BouncyCastle" Version="1.8.9" />
     <PackageReference Include="Dapper" Version="2.1.28" />
     <PackageReference Include="Ionic.Zip" Version="1.9.1.8" />
     <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
-    <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="7.0.11" />
+    <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="7.0.18" />
     <PackageReference Include="SSH.NET" Version="2024.0.0" />
     <PackageReference Include="System.Text.Encoding.CodePages" Version="8.0.0" />
     <PackageReference Include="System.Text.Json" Version="8.0.3" />
diff --git a/appsettings.Development.json b/appsettings.Development.json
index b122b44..8895671 100644
--- a/appsettings.Development.json
+++ b/appsettings.Development.json
@@ -16,7 +16,7 @@
     "AesIV": "7996209429192123"
   },
   "ConnectionStrings": {
-    "DBConnection": "Host=localhost;Database=new_cts;Username=postgres;Password=pgsql"
+    "DBConnection": "Host=postgres;Database=new_cts;Username=postgres;Password=pgsql"
   },
   "AllowedHosts": "*"
 }
